### PR TITLE
fix: app launch issue

### LINKED ISSUE #598: Blank screen on app launch
**Describe the bug**
On launching the app there is currently no fragment shown.

**To Reproduce**
Launch the app see the blank screen

**Expected behaviour**
On launching app create Badge Fragment should be seen 

**Screenshots**

ezgif com-video-to-gif (1)

**Smartphone Info:**

|               |   |
|---------------|---|
|Device         |Redmi 4A|
|Android Version|Nougat 7.1.2  |

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by refactor-droidyy**
Fixes #598 

Screenshots for the change:

ezgif com-video-to-gif (2)

### CODE REVIEW

**Comment by refactor-droidyy** [comment]
@adityastic please review.

**Comment by adityastic** [comment]
@refactor-droidyy give me 2 hrs. I'm on my way home, Will do in some time

**Comment by refactor-droidyy** [comment]
@adityastic no problem 👍

**Comment by refactor-droidyy** [comment]
@adityastic Done.

File: app/src/main/java/org/fossasia/badgemagic/ui/DrawerActivity.kt [hunk 1]
```diff
@@ -62,13 +62,22 @@ class DrawerActivity : BaseActivity(), NavigationView.OnNavigationItemSelectedLi
...
+
         setupDrawerAndToolbar()
 
         prepareForScan()
 
         handleIfReceiveIntent()
     }
 
+    private fun defaultFragment() {
+        if (supportFragmentManager.findFragmentById(R.id.frag_container) == null) {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/badgemagic/ui/DrawerActivity.kt]: Why is this needed?

Comment (refactor-droidyy) [reply app/src/main/java/org/fossasia/badgemagic/ui/DrawerActivity.kt]: @iamareebjamal initially there is no fragment so I have just set a default fragment if no fragment is there.

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/badgemagic/ui/DrawerActivity.kt]: Change method name to setDefaultFragment

**Pull request merged**